experimental/dds/tree2/src/class-tree/tree.ts

```diff
@@ -26,34 +26,35 @@ export interface ITree extends IChannel {
 	 * Returns a tree that is known to be compatible with the provided schema. The returned tree exposes an API that is schema-aware.
 	 *
 	 * @remarks
-	 * If the tree is uninitialized (has no schema and no content), the tree is initialized with the provided `initialTree` and `schema`.
+	 * If the tree is uninitialized (has no schema and no content), the tree is initialized with the `initialTree` and
+	 * view `schema` in the provided `config`.
```

Comment:
I'm not sure I get why we call it "view schema"; it seems to be used for more than just viewing purposes.

Reply:
@CraigMacomber